scene/gui/spin_box.cpp

```diff
@@ -78,6 +78,9 @@ void SpinBox::_text_submitted(const String &p_string) {
 }
 
 void SpinBox::_text_changed(const String &p_string) {
+	if (p_string.ends_with("."))
```

Comment:
```suggestion
	if (p_string.ends_with(".")) {
```

Reply:
👍 commit has been amended to add brackets.